crates/compiler/can/tests/test_suffixed.rs

```diff
@@ -5,16 +5,19 @@ extern crate indoc;
 mod suffixed_tests {
 
     use bumpalo::Bump;
+    use insta::assert_snapshot;
     use roc_can::desugar::desugar_defs_node_values;
+    use roc_parse::remove_spaces::RemoveSpaces;
     use roc_parse::test_helpers::parse_defs_with;
-    use roc_test_utils::assert_multiline_str_eq;
 
-    fn run_test(src: &str, expected: &str) {
-        let arena = &Bump::new();
-        let mut defs = parse_defs_with(arena, src).unwrap();
-        desugar_defs_node_values(arena, &mut defs, src, &mut None, "test.roc", true);
-        print!("{:#?}", &defs);
-        assert_multiline_str_eq!(format!("{:?}", &defs).as_str(), expected);
+    macro_rules! run_test {
+        ($src:expr) => {{
+            let arena = &Bump::new();
+            let mut defs = parse_defs_with(arena, $src).unwrap();
+            desugar_defs_node_values(arena, &mut defs, $src, &mut None, "test.roc", true);
+            print!("{:#?}", &defs);
+            assert_snapshot!(format!("{:#?}", &defs.remove_spaces(&arena)));
+        }};
```

Comment:
I like the use of snapshots, it is much easier to follow the AST that is produced

Reply:
There are two other key changes that make things easier: (1) using `{:#?}` instead of `{:?}` (i.e. pretty-printing), and (2) `remove_spaces`, which gets rid of a bunch of unhelpful clutter we don't need to check at this stage.

Reply:
Let's also import formatter so we can see the desugared roc code